src/pages/Staking/index.tsx

```diff
@@ -715,7 +733,20 @@ const Staking = (): JSX.Element => {
     submitButtonLabel = 'Loading...';
   } else {
     if (accountSet) {
-      submitButtonLabel = votingBalanceGreaterThanZero ? 'Add more stake' : 'Stake';
```

Comment:
Not something that needs to be fixed now (as with everything at the moment if it works, it's good enough!) but nested conditions like this are hard to read. All I'd ask is that you add a TODO here to refactor this the next time we're working on the staking feature,.

Reply:
@tomjeatt 
OK, let's improve it later. To be honest, I'm not sure how to improve it though. :)

Reply:
Thanks. Yes if it the solution was obvious it could be done now :) One of those problems that needs a bit of time to talk through.